### ISSUE TITLE
Cannot install by add DiffinDiffs and Documentation links (stable version) are invalid

### ISSUE THREAD

**Issue by duzaichuan** [JuliaDiffinDiffs/DiffinDiffs.jl#3]
Excited to know that such a useful package is now in Julia! As the title indicates, I found that two packages as a bundle are not available in project. Users have to install them separately and the doc link seems to be no longer valid.

And another issue w.r.t installing is that installation downgrades CSV from 0.10 to 0.8.5 and parsers.jl frome 2.2 to 1.1

**Comment by junyuan-chen** [comment]
Hi, thank you for your interest!

The component packages have been there for quite a while. But, I didn't register `DiffinDiffs.jl` until two days ago and I am still waiting for the initial registration, which will complete on the Julia package registry in several hours today.

The reason why the link for stable docs does not work is exactly because the initial registration has not been completed yet and hence no version number is recognized for being "stable".

For the dependencies, there are some planned changes regarding the internal organization of the packages. Once these reorganization is done, there will be updates that resolve these issues.
